tests/scripts/test_scripts.py

```diff
@@ -791,7 +791,7 @@ def test_parallel_arg_errors(tmpdir):
 
 
 def _generate_test_rollouts(tmpdir: str, env_named_config: str) -> pathlib.Path:
-    tmpdir_path = pathlib.Path(tmpdir)
+    tmpdir_path = types.parse_path(tmpdir)
```

Comment:
What was wrong with just using `pathlib.Path` here given `tmpdir` is guaranteed to be a `str`?

Reply:
Same as others.